synapse/lookup/pe.py

```diff
@@ -58,42 +65,78 @@ def getRsrcTypes():
     (0x1C01, 'ar-TN'),  # ARABIC_TUNISIA
     (0x3801, 'ar-AE'),  # ARABIC_UAE
     (0x2401, 'ar-YE'),  # ARABIC_YEMEN
+    (0x4401, "ar-Ploc-SA"),
```

Comment:
do we have any comments for what arabic dialects these are?

Reply:
My guess for those would be that those are "pseudolocales" since that's "ploc" seems to stand in for? It's not directly in the LCID docs. I had to hunt to find any description of it.

There are still a couple language tags like `ar-145` that I have no idea what they mean and have no references for, but microsoft included them in their standard :shrug:

Reply:
odd :shrug: